Dockerfile

```diff
@@ -25,23 +25,44 @@ RUN yum --assumeyes install \
     npm \
     openssl \
     procps-ng \
-    python-pip \
-    python3.7 \ 
+    python36 \
+    python39 \ 
+    python39-pip \
     rsync \
-    sshuttle \
-    the_silver_searcher \
-    tmux \
     vim-enhanced \
     wget \
-    && yum clean all;
+    && microdnf clean all;
+
 
+RUN git clone --depth 1 https://github.com/junegunn/fzf.git ~/.fzf && \
```

Comment:
`fzf` is installed (?) here via Github, but above via DNF.  Any reason why using one or the other?

Related: running ocm-container with these changes leaves me with an error on run:

```
[chris@marvin ocm-container]$ ocm-container
bash: /root/.fzf/shell/key-bindings.bash: No such file or directory
```

Reply:
Edit: ignore the install comment - I realize the DNF install was removed in favor of the github install.

The key-bindings.bash error comment still applies.

Reply:
I'll try fixing this issue :)

Reply:
WFM